Gandalf/build.sh

```diff
@@ -84,9 +129,13 @@ echo "Copying the DEBIAN scripts"
 
 cp "prerm" "${PKG_PACKAGE}/DEBIAN"
 cp "postinst" "${PKG_PACKAGE}/DEBIAN"
+echo
+read -p  "${BOLD}${RED}--- If you have anything else you need to put into Gandalf, now's the time. If you're finished or you don't want to add something press any key to continue. ---${NORMAL}"
+echo
+
+# Remove .DS_Store created by macOS (experimental)
 
-echo "If you have anything else you need to put into Gandalf, now's the time. You have 15 seconds from the moment this message appears."
-sleep 15
+find . -name '.DS_Store' -type f -delete
```

Comment:
Question you use `find . -type f -name '.DS_STORE' -exec rm {} +`
What's the + for?